src/components/AirbaseNG/AirbaseNG.tsx

```diff
@@ -1,165 +1,171 @@
                 handleProcessData("\nProcess completed successfully.");
-
-                // If the process was terminated manually, display a termination message.
             } else if (signal === 15) {
                 handleProcessData("\nProcess was manually terminated.");
-
-                // If the process was terminated with an error, display the exit and signal codes.
             } else {
                 handleProcessData(`\nProcess terminated with exit code: ${code} and signal code: ${signal}`);
             }
-
-            // Clear the child process pid reference. There is no longer a valid process running.
             setPid("");
-
-            // Cancel the loading overlay. The process has completed.
             setLoading(false);
-
-            // Allow Saving as the output is finalised
             setAllowSave(true);
             setHasSaved(false);
         },
-        [handleProcessData] // Dependency on the handleProcessData callback
+        [handleProcessData]
     );
 
     /**
-     * onSubmit: Asynchronous handler for the form submission event.
-     * It sets up and triggers the airbase-ng tool with the given parameters.
-     * Once the command is executed, the results or errors are displayed in the output.
-     *
-     * @param {FormValuesType} values - The form values, containing the fake host name, channel, and WLAN interface.
+     * Handles the form submission to run the airbase-ng command.
+     * @param {FormValuesType} values - Form values.
      */
     const onSubmit = async (values: FormValuesType) => {
-        // Activate loading state to indicate ongoing process
         setLoading(true);
-
-        // Disallow saving until the tool's execution is complete
         setAllowSave(false);
 
-        // Construct arguments for the aircrack-ng command based on form input
         const args = ["-e", values.fakeHost, "-c", values.channel, values.replayInterface];
 
-        values.macAddress ? args.push(`-a`, values.macAddress) : undefined;
-        values.filePath ? args.push(`-F`, values.filePath) : undefined;
-        values.customConfig ? args.push(values.customConfig) : undefined;
-
-        // Execute the aircrack-ng command via helper method and handle its output or potential errors
-        CommandHelper.runCommandWithPkexec("airbase-ng", args, handleProcessData, handleProcessTermination)
-            .then(({ output, pid }) => {
-                // Update the UI with the results from the executed command
-                setOutput(output);
-                setAllowSave(true);
-                console.log(pid);
-                setPid(pid);
-            })
-            .catch((error) => {
-                // Display any errors encountered during command execution
-                setOutput(error.message);
-                // Deactivate loading state
-                setLoading(false);
-                setAllowSave(true);
-            });
+        // Advanced Mode Options
```

Comment:
This is where some more descriptive commit messages would come in handy because I do not know why you changed this code. I find [this webpage](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) useful in understanding how to write better commit messages.